### PR TITLE
fix:  read stable coins list from a JSON file 

### LINKED ISSUE #2593: [Demand] Use built-in stablecoins list
## Catalogue

- [ ] UI/UX
- [ ] Mask Core Feature
- [x] Mask Plugin Feature
- [ ] Security/Cryptography Feature
- [ ] Other
  - /_List it here_/

## Part 1: Introduction and Background

The ERC20 stable coins list is not updated frequently. Let's built it into the plugin as a JSON file instead of fetching remotely.

## Part 2: Problem to Solve

Refactor `useStableTokensDebank` hook to read stable coins list from a JSON file.

### PR DESCRIPTION
**Pull request by lelenei**
close #2593

### CODE REVIEW

**Review by Jack-Works — DISMISSED** [review]
@guanbinrui plz review

File: packages/maskbook/src/web3/hooks/useStableTokensDebank.ts [hunk 1]
```diff
@@ -1,24 +1,14 @@
...
     const chainId = useChainId()
-    return useAsyncRetry(async () => {
-        const data = await fetcher(chainId)
-        return data.map((x: { id: string; decimals: number; name: string; symbol: string }) =>
-            createERC20Token(chainId, formatChecksumAddress(x.id), x.decimals, x.name, x.symbol),
-        )
-    }, [chainId])
+    if (chainId !== ChainId.Mainnet) return []
+
+    return StableCoins.data.map((x: { id: string; decimals: number; name: string; symbol: string }) =>
```
Comment (guanbinrui) [inline packages/maskbook/src/web3/hooks/useStableTokensDebank.ts:11]: The StableCoins can be a list of erc20records. Then you can use it directly.

**Review by guanbinrui — CHANGES_REQUESTED** [review]

File: packages/maskbook/src/web3/hooks/stables_coins.json [hunk 2]
```diff
@@ -0,0 +1,246 @@
+{
```
Comment (guanbinrui) [inline packages/maskbook/src/web3/hooks/stables_coins.json:1]: Do not copy & paste the response. Make it an array of `ERC20TokenDetailed` entities.

**Review by Jack-Works — APPROVED** [review]
LGTM on style

File: packages/maskbook/src/web3/hooks/stables_coins.json [hunk 3]
```diff
@@ -0,0 +1,240 @@
+[
+    {
+        "active_user_count_24h": 99504,
```
Comment (guanbinrui) [inline packages/maskbook/src/web3/hooks/stables_coins.json:3]: Please remove below fields :

+ active_user_count_24h
+ total_supply
+ transfer_amount_24h
+ transfer_count_24h

**Review by guanbinrui — APPROVED** [review]

**Review by Jack-Works — APPROVED** [review]

**Comment by lelenei** [comment]
close

**Pull request merged**